### DISCUSSION TITLE
Account suspended and support ignores me

### DISCUSSION THREAD

**Discussion by DiorFlipFlops** [community/community#48787] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

On February 3rd, I was testing my project using codespaces when suddenly, I was hit by the message "Account suspended. Access to your account has been suspended due to a violation of our Terms of Service.",
This may have been because I was sending a significant amount of HTTP requests to test some stuff,
so, of course, I sent an appeal right away,

now the weird thing is that they haven't responded for almost a month which makes me think something in the appeal system is broken.

I would love to know if you guys can escalate the appeal or do anything about it; my original account is github.com/fknMega

thank you

**Comment by dmilburn** [comment]


**Reply by cunzhang2** [reply]
Hi there @dmilburn, On January 30rd, my friend's account @zonemeen was suspended without any notice, then he immediately sent a ticket to support, but now it's been more than a month and haven't received any reply. Can you help him to receive a reply as soon as possible? The ticket id is: 1989881. Thank you very much.❤️

**Reply by Awesome-GJK** [reply]
This Monday, I found out that my github account was banned, but I didn't receive any email reminder. Since Monday, I have filed complaints in github support every day, but there is no response, no official contact me.

Since I used github, I have never done any illegal operations, strictly abide by the github account usage rules, I don't know why the official banned me, I don't understand!

I hope the official will contact me as soon as possible to help me unblock my account, thanks.

**Reply by Awesome-GJK** [reply]
my ticket : 2046081 Thanks

**Reply by ghost** [reply]
Same problem here. I was testing GitHub Classroom and CodeSpaces with my thesis advisor to implement them in the undergraduate courses when my account @nextmarte was suspended. I opened an appeal but received no response. If you could help, I would appreciate it.

Obs.: I did not received any e-mail or warning regarding the suspension.

**Reply by matthewmartinhughes** [reply]
I have had similar issues on my corporate or enterprise account and the problem seems to be the hackers or guilty parties have already gotten to your contact information and the telecommunications databases.  They reroute the responses from any submissions or overwhelm users accounts with spam and other techniques.

**Reply by ghost** [reply]
Same here :)

**Reply by ghost** [reply]
Same here got banned for no reason

**Comment by DiorFlipFlops** [comment]
> Hi from the Codespaces team, I've escalated internally and will hopefully get you a response soon. Sorry for the delay

Are you sure there's still no update

**Reply by dmilburn** [reply]
Hi! Sorry I thought you'd receive an email and didn't need a notice from me. Your account has been re-enabled!

**Reply by ghost** [reply]
Hi @dmilburn  and @adnamalin ,
My Github account: https://github.com/SayedAlAhlMohamed
was suspended.
I don't know why as I didn't receive any mails or hints indicating why this happened.
I was taking to some one here in the community trying to solve a problem when suddenly it logged me out and when tried to sign in a message appear that account suspended for violating GitHub’s terms of service.Can some one please what is happening?
I am so sad and can't describe my sadness. I never violated TOS in my life and if that happened it is absolutely a mistake and not intentional. I’m happy to oblige with all of GitHub’s policies and Terms.

Thank you!

**Comment by cheese1218** [comment]
Hey there @dmilburn and @adnamalin 
Hope you two are both having a wonderful day! 
My github account https://github.com/cheese529 was also suspended randomly out of the blue without any email, I was submitting a couple bugs and randomly got logged out and upon trying to sign back in it said my account had been suspended for violating ToS. The devs of the project I was submitting the bugs for told me that my account could have been falsely flagged for spam since I submitted 3 issues fairly quickly and they told me to appeal the suspension and said they would be more than happy to help vouch for me as well.

I have never done anything to violate the terms of use in anyway at all so I truly believe this was a false ban, I have also submitted a ticket twice but I have heard no reply back. This is so frustrating because I need my account for work and I have no idea what to do, I need it to be reinstated as soon as possible, please help me out here. I’m happy member of github and would love to continue to follow and oblige with all of GitHub’s policies and Terms please if you could help me ASAP it would truly mean the world to me. My ticket id- 2043929

**Reply by dmilburn** [reply]
Hi there!

We're not able to discuss account specifics on here so if you've not already done so please raise a [Reinstatement Request](https://support.github.com/contact/reinstatement)

If you have already raised this please be patient and someone will get back to you as soon as they are able with any information etc. Submitting more requests will only slow any response.

You should note that the GitHub [Terms of Service](https://docs.github.com/github/site-policy/github-terms-of-service#3-account-requirements) state that an individual may not maintain more than one free user account, even if the one you have is restricted in some way, so this additional account may also find itself restricted in future.

As I say, please exercise patience and someone will be in touch on the email address you fill in on the reinstatement form.

**Reply by cheese1218** [reply]
Please understand I needed to communicate with certain developers for my business so I am losing money everyday that my account is banned. I already raised a reinstatement request and it has been a month now, nobody has replied to me. I also understand it is against the terms of service to not have more than one free user account and that is fine by all means, I can delete this account once they unban my main. Please if it is possible for you to escalate my claim somehow like you helped the that other gentleman it would mean alot to me.

**Reply by ghost** [reply]
Hi @dmilburn,

I raised a ticket  3 months ago and they never answered me!! so I made another account and they just suspended it after one month. I didn't know about TOS having multiple accounts. now when I do, I asked them to delete all my accounts and recover
this one only: https://github.com/Sayedalahlmohamed
I am happy to oblige with all github Terms and conditions , but please help us here!!
we must all cooperate here!!
here are my ticket IDs: #1977213
and the new one: #2049915

**Comment by montwall** [comment]
Hi, @dmilburn , yesterday (10/04/2023), my friend's account @richman225 was suspended without any notice and he sent a ticket to support immediately since he is working with his client on the project at the moment, but not received any reply yet. Can you help him to receive a reply as soon as possible? The ticket id is: 2359209 and 2359069. He really hope can get the account recovered asap!

**Comment by ghost** [comment]
Hi, I've been banned on github for no reason.

I did not receive any warning or mail linked to that.

My ticket is 2464386 
The account was @sirlavoie I got banned (12/3/2023)

I really need my account back for my school project and everything...

**Comment by LouisWinkler** [comment]
Hi @dmilburn, 
I wanted to bring to your attention that my friend's account @seniorcoder72 was suspended on Dec 13 without any prior notice. 
He immediately sent a support ticket but hasn't received a response yet. 
He is currently working on a project with a client and is in need of urgent assistance. 
Could you please help him receive a prompt response? 
The ticket ID is 2480585. 
He is hoping to have his account recovered as soon as possible. 
Really he wants it. He thinks he has been followed ToS, but he has been suspended. Can you help him?
Please help.
Thank you.

**Comment by JhoanaEspana** [comment]
@dmilburn  My account has also been suspended, what should I do, I need my repositories and my projects, urgently!!!

I cannot upload or download repositories when doing so I get this error "You are marked as spam, and therefore cannot authorize a third party application"
how can I solve that?

ID ticket: 2492056

![Captura de pantalla 2023-12-20 225445](https://github.com/community/community/assets/82187479/e5976110-943f-45e0-a381-fa533ebf057f)

**Reply by abhishekHegde2000** [reply]
Hi , my GitHub account also got marked as spam, can you please tell me if you account got reinstated or not,
it very urgent , if yes please tell me how were you able to do that?
i raised support ticket but not getting any replies
@JhoanaEspana

**Comment by InsipidLake** [comment]
Hey @dmilburn , My account has also been suspended. My account was hacked, or logged in from somewhere suspisious and github has sent me an email to force a password reset, but right after resetting the password my account got suspended. I really need my repositories.

ID ticked: 2504785

**Comment by ahegde28** [comment]
@dmilburn 
My main (abhishekHegde2000) account has also been suspended, what should, its showing as account flagged.
please i need you help.

my account is not visible to anyone.
so i created second account to get into discussion, please help.

ticket id: #2503835

please help me, its very urgent

**Comment by abhishekHegde2000** [comment]
Hi , my GitHub account also got marked as spam, can you please tell me if you account got reinstated or not,
it very urgent , if yes please tell me how were you able to do that?
i raised support ticket but not getting any replies
@JhoanaEspana

**Comment by abhishekHegde2000** [comment]
Hi , my GitHub account also got marked as spam, can you please tell me if you account got reinstated or not,
it very urgent , if yes please tell me how were you able to do that?
i raised support ticket but not getting any replies
@dmilburn

**Comment by ramgoel1** [comment]
@dmilburn My github account was suspended yesterday. I haven't received any warning or email. It has everything whatever I have done in tech till date. I request you to please assist me in this matter.

Deleted Account Username - RamGoel
Ticket ID - 2508635

**Comment by huazidev2** [comment]
Hi there @dmilburn , my account @huazidev was suspended five days ago without any notice, i immediately sent a ticket to support with some possible reason, but now it haven't received any reply. Can you help me to receive a reply as soon as possible? Thank you very much.❤️

Suspended account: huazidev
The ticket id is: 2509981.

**Comment by sussahoo** [comment]
This is so annoying, I just started with the copilot trial, and suddenly my account is suspended and I can't even opt out of the copilot trial as it doesn't allow me to log in. 


Ticket ID: 2509310
Suspended account:  4eversusanta

**Comment by kish191919** [comment]
Hi my account was suspended last week ago and I immediately sent a ticket to support team.
But I have not received any reply. Would anyone in Git please help me to resolve this issue?

Ticket ID: 2514147
Suspended account : DreamHighData

**Comment by aletsmtz** [comment]
Hi! My account was suspended right after creating it

Ticket ID: 2519439
Suspended account : aletshobops or alejandro.martinez@hobops.io

**Comment by ossanna16** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
